docs/content/guide/decorators.ngdoc

```diff
@@ -0,0 +1,475 @@
+@ngdoc overview
+@name Decorators
+@sortOrder 345
+@description
+
+# Decorators in AngularJS
+
+<div class="alert alert-warning">
+  **NOTE:** This guide is targeted towards developers who are already familiar with AngularJS basics.
+  If you're just getting started, we recommend the {@link tutorial/ tutorial} first.
+</div>
+
+## What are decorators?
+
+Decorators are used to facilitate a decorator design pattern. This pattern is used to separate modification or
```

Comment:
Maybe "Decorators are a design pattern that is used to separate modification or decoration of a class without modifying the original source code" There's currently a lot of repetition here.